packages/react-instantsearch-core/src/connectors/__tests__/connectMenu.js

```diff
@@ -424,16 +424,16 @@ describe('connectMenu', () => {
       };
       results.getFacetValues.mockClear();
       results.getFacetValues.mockImplementation(() => [
-        {
-          name: 'wat',
-          isRefined: false,
-          count: 20,
-        },
         {
           name: 'oy',
           isRefined: true,
           count: 10,
         },
+        {
+          name: 'wat',
+          isRefined: false,
+          count: 20,
+        },
```

Comment:
I guess since now we no longer "order the facets by `isRefined`" but rather let getFacetValues handle it, the description of the test should change to something that "uses the getFacetValue default order.
And check that we're passing it `undefined` when `searchable && !isFromSearch`.

Reply:
Makes sense, I’ll change the test description

Reply:
You forget to change the name of the test?